docs/source/en/optimization/memory.md

```diff
@@ -295,13 +301,6 @@ pipeline.transformer.enable_group_offload(onload_device=onload_device, offload_d
 
 The `low_cpu_mem_usage` parameter can be set to `True` to reduce CPU memory usage when using streams during group offloading. It is best for `leaf_level` offloading and when CPU memory is bottlenecked. Memory is saved by creating pinned tensors on the fly instead of pre-pinning them. However, this may increase overall execution time.
 
-<Tip>
```

Comment:
Why is this moving away?

Reply:
I moved this `[!TIP]` to the start of the `speed-memory-optims.md` guide to keep the `memory.md` guide more focused.

There is a link at the beginning of the `## Offloading` section to `speed-memory-optims.md` so users aren't missing out on anything and I think it'll also actually increase visibility!